internal/driver/cli.go

```diff
@@ -58,6 +59,7 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagTools := flag.String("tools", os.Getenv("PPROF_TOOLS"), "Path for object tool pathnames")
 
 	flagTimeout := flag.Int("timeout", -1, "Timeout in seconds for fetching a profile")
+	flagHTTPPort := flag.Int("http", 0, "Present interactive web based UI at the specified http port")
```

Comment:
1. I think reusing the name "web" will be problematic since it will change the behavior substantially without warning.  E.g., consider somebody who has a script that does "pprof web" a few times to open a bunch of views in a row.  The interactive UI will block on the first call.  I also am not so excited about "There's always "svg" for people that really want a static svg."  If we do that, we are going to force some people to learn a new behavior after they get confused a bit about why things don't work as they used to; and I don't think the win here is big enough to justify that.

2. W.r.t. putting "http" under "Output Formats", I still think that it is better to keep things as they are for the reasons mentioned above (usage message, port number not a string, interactive command misuse).

So unless I hear a stronger complaint than "I'm OK either way", I will keep "http" out of Output Formats.